src/main/java/com/exasol/sql/expression/rendering/ValueExpressionRenderer.java

```diff
@@ -1,127 +1,239 @@
+     * 
+     * @param config render configuration
+     */
     public ValueExpressionRenderer(final StringRendererConfig config) {
         super(config);
     }
 
     @Override
-    public void visit(final StringLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append("'");
-        append(literal.toString());
-        append("'");
-        setLastVisited(literal);
+    public void visit(final Not not) {
+        appendKeyword("NOT");
+        startParenthesis();
+        not.getOperand().accept((BooleanExpressionVisitor) this);
+        endParenthesis();
     }
 
     @Override
-    public void visit(final IntegerLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append(literal.toString());
-        setLastVisited(literal);
+    public void visit(final And and) {
+        startParenthesisIfNested();
+        append(" AND ", and.getOperands());
+        endParenthesisIfNested();
+    }
+
+    private void append(final String separator, final List<BooleanExpression> expressions) {
+        boolean isFirst = true;
+        for (final BooleanExpression expression : expressions) {
+            if (!isFirst) {
+                appendKeyword(separator);
+            }
+            isFirst = false;
+            expression.accept((BooleanExpressionVisitor) this);
+        }
     }
 
     @Override
-    public void visit(final LongLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append(literal.toString());
-        setLastVisited(literal);
+    public void visit(final Or or) {
+        startParenthesisIfNested();
+        append(" OR ", or.getOperands());
+        endParenthesisIfNested();
     }
 
     @Override
-    public void visit(final DoubleLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append(literal.toString());
-        setLastVisited(literal);
+    public void visit(final BooleanLiteral literal) {
+        appendBooleanLiteral(literal);
     }
 
     @Override
-    public void visit(final FloatLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append(literal.toString());
-        setLastVisited(literal);
+    public void visit(final Comparison comparison) {
+        comparison.accept((ComparisonVisitor) this);
     }
 
+    /*
+     * Comparison renderer
+     */
     @Override
-    public void visit(final BigDecimalLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append(literal.toString());
-        setLastVisited(literal);
+    public void visit(final SimpleComparison simpleComparison) {
+        openComparison(simpleComparison);
+        closeComparison();
     }
 
     @Override
-    public void visit(final BooleanLiteral literal) {
-        appendCommaWhenNeeded(literal);
-        append(literal.toString());
-        setLastVisited(literal);
+    public void visit(final LikeComparison like) {
+        openComparison(like);
+        if (like.hasEscape()) {
+            this.builder.append(" ESCAPE ");
```

Comment:
```suggestion
            this.builder.appendKeyword(" ESCAPE ");
```

Mind the rendering config settings! Please check for other instance of this problem too.